website/docs/sql-reference/string-functions/sql-trim.md

```diff
@@ -0,0 +1,53 @@
+---
+id: trim
+title: SQL TRIM
+description: The SQL TRIM function removes the leading and trailing characters of a string. By default, it removes the blank space character from the beginning and end of a string.
+slug: /sql-reference/trim
+---
+
+<head>
+    <title>Working with the SQL TRIM function</title>
+</head>
+
+We’ve been there: pesky blank spaces, weird, inconsistent formats, or unaccountable asterisks hiding at the end of your column value—strings are one of the most variable data types in your datasets. They likely lack a uniform casing, vary in length, and will inevitably have characters you need to trim from them.
+
+Introducing: the SQL TRIM function, which removes the leading and trailing characters of a string. By default, it removes the blank space character from the beginning and end of a string.
+
+## How to use the SQL TRIM function
+
+The syntax for using TRIM function looks like the following:
+
+```sql
+trim(<field_name> [, <characters_to_remove>])
+```
+
+Like we said earlier, the default `<characters_to_remove>` is a blank space, such that if you were to `trim(' string with extra leading space')` it would return 'string with extra leading space'`. You can explicitly specify single characters or a pattern to trim from your strings.
```

Comment:
```suggestion
Like we said earlier, the default `<characters_to_remove>` is a blank space, such that if you were to `trim(' string with extra leading space')` it would return `'string with extra leading space'`. You can explicitly specify single characters or a pattern to trim from your strings.
```